source/_docs/sftp.md

```diff
@@ -7,37 +7,33 @@ keywords: sftp, sftp mode, wp-admin, apps.module, administrator, admin, connecti
 ---
 In some cases, working via Git is not the best option. You may not like local development, or you may want to show work to a remote collaborator (or client) immediately, or need to debug a specific problem that only occurs on the Pantheon platform.
 
-This is why we offer **SFTP mode**. This allows you to develop directly on Pantheon and can be a major time-saver.
+SFTP mode allows you to develop directly on Pantheon and can be a major time-saver. If you want to [use your website framework's built-in update systems](/docs/more-sftp/) (e.g. the <code>apps.module</code> in Drupal, or the plugin/theme manager in WordPress), enable SFTP first.
 
-If you want to use your website framework's built-in update systems (e.g. the <code>apps.module</code> in Drupal, or the plugin/theme manager in WordPress), enable SFTP first.
+<div class="alert alert-danger" role="alert"><h4>SFTP Mode Considerations: </h4>
+<ul>
+<li> You need to <strong>commit frequently</strong>.

```

Comment:
Why? Are all of the remaining bullet points the "why"  Can we make that more clear with formatting or maybe adding... "because of the following:"